crates/accelerate/src/sparse_observable/mod.rs

```diff
@@ -1263,6 +1645,24 @@ impl From<ArithmeticError> for PyErr {
     }
 }
 
+/// The single-character string label used to represent this term in the :class:`SparseObservable`
+/// alphabet.
+#[pyfunction]
+#[pyo3(name = "label")]
+fn bit_term_label(py: Python, slf: BitTerm) -> &Bound<PyString> {
```

Comment:
What's the difference of this to `BitTerm.py_label`?

Reply:
it's in the comment 1 line before this diff haha:

> This doesn't use `py_label` so we can use `intern!`.

Reply:
Ahh, this reads as "we would like to use `intern!` hence we don't use `py_label`"